src/ducks/settings/CategoryAlerts/CategoryAlertSettingsPane.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react'
+import compose from 'lodash/flowRight'
+import { withClient, queryConnect } from 'cozy-client'
+import { Button, Stack, Alerter, translate } from 'cozy-ui/transpiled/react'
+
+import TogglePane, {
+  TogglePaneTitle,
+  TogglePaneText
+} from 'ducks/settings/TogglePane'
+
+import { settingsConn } from 'doctypes'
+import { getDefaultedSettingsFromCollection } from 'ducks/settings/helpers'
+
+import CategoryAlertCard from 'ducks/settings/CategoryAlerts/CategoryAlertCard'
+import CategoryAlertEditModal from 'ducks/settings/CategoryAlerts/CategoryAlertEditModal'
+
+const makeNewAlert = () => ({
+  categoryId: '400100',
+  balanceThresoldMin: 100,
+})
+
+const CreateCategoryAlert = translate()(({ createAlert, t, ...props }) => {
+  const [creating, setCreating] = useState(false)
+  const [saving, setSaving] = useState(false)
+  const handleCreateAlert = async newAlert => {
+    setCreating(false)
+    try {
+      setSaving(true)
+      await createAlert(newAlert)
+    } finally {
+      setSaving(false)
+    }
+  }
+  return (
+    <>
+      <Button
+        className="u-ml-0"
+        theme="secondary"
+        label={t('Settings.budget-category-alerts.create-alert')}
+        busy={saving}
+        onClick={() => {
+          setCreating(true)
+        }}
+        {...props}
+      />
+      {creating ? (
+        <CategoryAlertEditModal
+          onDismiss={() => setCreating(false)}
+          initialAlert={makeNewAlert()}
+          onEditAlert={handleCreateAlert}
+        />
+      ) : null}
+    </>
+  )
+})
+
+const Grid = ({ children }) => <div>{children}</div>
+
+/**
+ * Replace item in arr, finding item through idFn.
+ * If previous item cannot be found, the new item is pushed at the
+ * end of the array.
+ */
+const replaceBy = (arr, item, idFn) => {
+  const id = idFn(item)
+  const index = arr.findIndex(x => idFn(x) === id)
+  return index !== -1
+    ? [...arr.slice(0, index), item, ...arr.slice(index + 1)]
+    : [...arr, item]
+}
+
+const maxValue = (arr, fn) => {
+  const rs = arr.reduce((curMax, item) => {
+    const v = fn(item)
+    return typeof v === 'number' ? Math.max(v, curMax) : curMax
+  }, 0)
+  return rs
+}
```

Comment:
Isn't Lodash's maxBy doing the same job?

Reply:
maxBy will return the item, not the value. But I agree, it is simpler to use maxBy, I changed the code to use it.